libs/datasets/sources/nytimes_dataset.py

```diff
@@ -48,6 +63,22 @@ def local(cls) -> "NYTimesDataset":
         data_root = dataset_utils.LOCAL_PUBLIC_DATA_PATH
         return cls(data_root / cls.DATA_FOLDER / cls.COUNTIES_DATA_FILE)
 
+    @classmethod
+    def _remove_ct_cases(cls, data: pd.DataFrame) -> pd.DataFrame:
+        is_county = data[cls.Fields.AGGREGATE_LEVEL] == "county"
+        is_ct = data[cls.Fields.STATE] == "CT"
+        # July 24th was the day there was a backfill of cases
+        is_on_or_after_july_24 = data[cls.Fields.DATE] >= "2020-07-24"
+        is_ct_county_data_after_jul_24 = is_county & is_ct & is_on_or_after_july_24
+
+        # Remove backfilled tests from cumulatives on and after July 24th.
+        for fips, count in ESTIMATED_REMOVED_CT_CASES_COUNTY.items():
+            is_ct_fips_data_after_jul_24 = is_ct_county_data_after_jul_24 & (
+                data[CommonFields.FIPS] == fips
```

Comment:
how about matching on FIPS and DATE only? I'd like to reduce our dependency on other fields so we can more easily support other region types. I like project Li locationID.

Reply:
Oh do you mean filtering out the ct filter? that makes sense to me

Reply:
Yes, you got it.